Firestore/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 - [fixed] Fixed a regression in the Firebase iOS SDK release 4.11.0 that could
   cause `getDocument()` requests made while offline to be delayed by up to 10
   seconds (rather than returning from cache immediately).
+- [feature] Added a new `Timestamp` class to represent timestamp fields,
+  currently supporting up to microsecond precision. It can be passed to API
+  methods anywhere a system Date is currently accepted. To make
+  `DocumentSnapshot`s read timestamp fields back as `Timestamp`s instead of
+  Dates, you can set the newly-added flag `areTimestampsInSnapshotsEnabled` in
```

Comment:
I believe the setter is actually called `setTimestampsInSnapshotsEnabled`. It's exposed as a property though so it's also accessible as just `timestampsInSnapshotsEnabled`.

Therefore I think you want this phrase to be

To make ..., you can set the newly added `timestampsInSnapshotsEnabled` property in `FirestoreSettings` to `true`.

Please add a swift compilation test to verify that this is indeed true.

Reply:
Hmm, I don't know anything about Swift, but this doesn't seem to work:
* `timestampsInSnapshotsEnabled` gives _"'timestampsInSnapshotsEnabled' has been renamed to 'areTimestampsInSnapshotsEnabled'. 'timestampsInSnapshotsEnabled' was obsoleted in Swift 3"_;
* `setTimestampsInSnapshotsEnabled` gives _"Value of type `FirestoreSettings` has no member `setTimestampsInSnapshots`.

`areTimestampsInSnapshotsEnabled` is the only one that seems to work (note that the existing Swift build test also has `settings.isPersistenceEnabled = true`). Added this line to Swift compilation test (and also instantiation of `Timestamp` in `types` function). Removed the hyphen in "newly-added" and replaced "flag" with "property" in changelog entry.

Reply:
Ah right. Sorry about that. In Objective-C and early versions of Swift the property could have a different getter and setter name. The test shows you're definitely right. Thanks!